inst/rmarkdown/templates/inbo_boek/skeleton/_bookdown.yml

```diff
@@ -1,2 +1,18 @@
 book_filename: "INBO-boek.Rmd"
 new_session: FALSE
+rmd_files:   # specifies the order of Rmd files; NOT needed when you use index.Rmd and an alphabetical order for other Rmd files
+#    - 1stFileOfBook.Rmd (if you use a file 'index.Rmd', it should be here)
+#    - 2ndFileOfBook.Rmd
+#    - 3rdFileOfBook.Rmd
+#    - <and so on>
+language: 
+  label:  # comment out the below parameters if you wish English in output formats, other than bookdown::pdf_book
```

Comment:
I don't think that this is required. At least fig and tab are handled by the latex style